integrationtest/java/modules/cdi/pom.xml

```diff
@@ -24,13 +24,14 @@
     <properties>
         <surefire.jvm.args.additional>-Duser.language=en -Duser.country=US</surefire.jvm.args.additional>
         <!--
-           Caused by: java.lang.IllegalAccessError: superinterface check failed:
-           class org.hibernate.validator.internal.engine.ValidatorFactoryImpl$Proxy$_$$_WeldClientProxy
-           (in module org.hibernate.validator) cannot access class org.jboss.weld.interceptor.util.proxy.TargetInstanceProxy
-           (in unnamed module @0x6d763516) because module org.hibernate.validator does not read unnamed module @0x6d763516
+          If we do not read all unnamed for jboss logging it'll lead to:
+            Caused by: java.lang.IllegalArgumentException: This library does not have private access to interface org.jboss.weld.logging.VersionLogger
+                at org.jboss.logging@3.6.0.Final/org.jboss.logging.Logger.getMessageLogger(Logger.java:2569)
+                at org.jboss.logging@3.6.0.Final/org.jboss.logging.Logger.getMessageLogger(Logger.java:2549)
+                at org.jboss.weld.logging.VersionLogger.<clinit>(VersionLogger.java:35)
```

Comment:
Shouldn't Weld be in the module path in your test?

Reply:
this particular exception is something from jboss-logging itself, there's a check for modules and since current version of Weld is on an older JBoss logging things fail.

As to where Weld should be .... IDK 🫨 😃. My understanding was that we should only have the `jakarta.cdi` as required in the module definition, and then Weld is just added as a test dependency... no?

Reply:
> this particular exception is something from jboss-logging itself, there's a check for modules and since current version of Weld is on an older JBoss logging things fail.

ok :shrug: